speckle/resources/objects.py

```diff
@@ -24,15 +24,14 @@ class SpeckleObject(ResourceBaseSchema):
     ancestors: Optional[List[str]]
 
     
-    def dict(self, include=None, exclude=None, by_alias=False, exclude_unset=False, exclude_defaults=False, exclude_none=False):
+    def dict(self, include=None, exclude=None, by_alias=True, exclude_unset=False, exclude_defaults=False, exclude_none=False):
         json_string = json.dumps(super(SpeckleObject, self).dict()['properties'])
 
-        self.geometryHash = hashlib.md5(
-            json_string.encode('utf-8')).hexdigest()
+        self.geometryHash = hashlib.md5(json_string.encode('utf-8')).hexdigest()
 
-        self.hash = hashlib.md5('{}.{}'.format(self.type, json_string).encode('utf-8')).hexdigest()
+        #self.hash = hashlib.md5('{}.{}'.format(self.type, json_string).encode('utf-8')).hexdigest()
```

Comment:
Why are we removing this?

Reply:
Maybe it was just in my testing, but when trying to create an object on the server with the same geometry + name, it generated the same hash code, which prevented the new object being created, and hence to some head scratching when the object wouldn't update. Not 100% it's unnecessary... I'm happy to uncomment it?

Reply:
Cool. So long as you're happy with the fact that this is an issue then that's ok. I forget where speckle handles hashing (if its on the client or server side...)

Reply:
This is re-instated.

Reply:
Ah, well, let's see how it goes... if it becomes a problem, I'll comment it out again and make a new PR 😄

Reply:
Argh... re-commented this. The hash string is based on the object properties, which don't exist for many objects, so we get the same hash for many objects, which ends up return meaningless object IDs.

Reply:
Right that makes sense. I think you're right, we should comment it out.